### PR TITLE
Adds Pickle-Coding to atmospherics and power CODEOWNERS.

### PR DESCRIPTION
**Pull request by Pickle-Coding**
So I get notified.

### CODE REVIEW

**Review by LemonInTheDark — COMMENTED** [review]
Poor fool

**Comment by LemonInTheDark** [comment]
Oh right pickle is insane

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -126,6 +126,13 @@
 /code/modules/capture_the_flag/ @NamelessFairy
 /_maps/map_files/CTF/ @NamelessFairy
 
+# Pickle-Coding
+
+/code/__DEFINES/atmospherics/ @Pickle-Coding
+/code/modules/atmospherics/ @Pickle-Coding
+/code/modules/power/ @Pickle-Coding
+/code/__DEFINES/reactions.dm @Pickle-Coding
```
Comment (LemonInTheDark) [inline .github/CODEOWNERS:134]: This should be alphabetized. Defines come before everything else
yes I will be pranking ryll for not following the same pattern

**Comment by Pickle-Coding** [comment]
Done.

**Comment by optimumtact** [comment]
Screenshot from 2022-03-09 14-10-22

**Pull request merged**